go.mod

```diff
@@ -5,27 +5,27 @@ go 1.18
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v1.2.4
-	github.com/onsi/ginkgo/v2 v2.9.5
-	github.com/onsi/gomega v1.27.7
-	github.com/openshift/api v0.0.0-20230804173756-26b8597c4de2
-	github.com/openshift/client-go v0.0.0-20230503144108-75015d2347cb
-	github.com/openshift/library-go v0.0.0-20230508110756-9b7abe2c9cbf
-	github.com/openshift/machine-api-operator v0.2.1-0.20230524144558-2430daf07c93
-	golang.org/x/oauth2 v0.6.0
-	google.golang.org/api v0.107.0
-	k8s.io/api v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/client-go v0.27.2
+	github.com/onsi/ginkgo/v2 v2.12.1
+	github.com/onsi/gomega v1.28.0
+	github.com/openshift/api v0.0.0-20230928105710-23b54c280f99
+	github.com/openshift/client-go v0.0.0-20230926161409-848405da69e1
+	github.com/openshift/library-go v0.0.0-20230927113136-405c34317fa4
+	github.com/openshift/machine-api-operator v0.2.1-0.20230929171041-2cc7fcf262f3
+	golang.org/x/oauth2 v0.8.0
+	google.golang.org/api v0.114.0
+	k8s.io/api v0.28.2
```

Comment:
You probably need to rebase as we updated all the deps to 1.28 in another PR recently